frontend/components/ProductCard.vue

```diff
@@ -148,9 +148,12 @@ function placeBid() {
   console.log("value", bid.value);
   try {
     bidWait.value = true;
-    bidEncrypted(props.Product.contract, bid.value).then((result) => { 
+    bidEncrypted(props.Product.contract, bid.value).then((result) => {
       showBid.value = false; 
       bidWait.value = false;
+      getMyBid(props.Product.contract).then(result => {
+        myBidLocal.value = result;
```

Comment:
this solves an issue where the bid does not show up for the bidder if he doesn't refresh the page.